Email_Scheduler.py

```diff
@@ -0,0 +1,56 @@
+import smtplib, ssl
+from email.mime.multipart  import MIMEMultipart
+from email.mime.text  import MIMEText
+from datetime import datetime
+import time
+
+
+def send_email(smtp_server, port, sender_email, password, receiver_email, subject, body):
+    msg=MIMEMultipart()
+    msg['From']=sender_email
+    msg['To']=receiver_email
+    msg['Subject']=subject
+    msg.attach(MIMEText(body,'html'))
+
+    server=smtplib.SMTP(smtp_server,port)
+    server.starttls()
+    server.login(sender_email,password)
+    server.send_message(msg)
+    server.quit()
+
+server = 'smtp.gmail.com'
+port=587
```

Comment:
Will this work on every pc?!

Reply:
Yes it will if an env is setup for python

Reply:
Port i am asking.

Reply:
Yes, if it is being sent to gmail.com it will work. If it is something other like outlook or yahoo, it might not

Reply:
Okay.